harpy/_validations.py

```
@@ -300,82 +301,92 @@
+                    print_error("invalid format", f"The input file is formatted incorrectly at line {idx}. This is the first row triggering this error, but it may not be the only one.")
+                    print_solution_with_culprits(
+                        f"Rows in [blue]{regioninput}[/] need to be [bold]space[/] or [bold]tab[/] delimited with the format [yellow bold]contig start end[/] where [yellow bold]start[/] and [yellow bold]end[/] are integers.",
+                        "Rows triggering this error:"
+                        )
+                    click.echo(line, file = sys.stderr)
+                    sys.exit(1)
+                else:
+                    try:
+                        start = int(row[1])
+                        end = int(row[2])
+                    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)